forms.html

```diff
@@ -0,0 +1,119 @@
+            </div>
+          </div>
+        </div>
+        <div id="show_error"></div>
+        <div class="field is-horizontal">
+          <div class="field-label is-normal">
+            <label class="label">Email:</label>
+          </div>
+          <div class="field-body">
+            <div class="field">
+              <div class="control">
+                <input class="input" type="email" placeholder="email" id="email" name="email">
+              </div>
+            </div>
+          </div>
+        </div>
+        <div id="show-error"></div>
+        <div class="field is-horizontal">
+          <div class="field-label is-normal">
+            <label class="label">Password:</label>
+          </div>
+          <div class="field-body">
+            <div class="field">
+              <div class="control">
+                <input class="input" type="number" placeholder="mobile number" id="number" name="number">
+              </div>
+            </div>
+          </div>
+        </div>
+        <div id="showerror"></div>
+        <div class="field is-horizontal">
+          <div class="field-label is-normal">
+            <label class="label">Message:</label>
+          </div>
+          <div class="field-body">
+            <div class="field">
+              <div class="control">
+                <textarea class="textarea" placeholder="Explain how we can help you" id="textarea" name="textarea"></textarea>
+              </div>
+            </div>
+          </div>
+        </div>  
+        <div id="errorshow"></div>     
+        <div class="field is-horizontal">
+          <div class="field-label">
+            <label class="label">Gender:</label>
+          </div>
+          <div class="field-body">
+            <div class="field is-narrow">
+              <div class="control">
+                <label class="radio">
+                  <input type="radio" name="member" value="male" id="select" name="radioButton">
+                  male
+                </label>
+                <label class="radio">
+                  <input type="radio" name="member" value="female" id="select" name="radioButton">
+                  female
+                </label>
+                <label class="radio">
+                  <input type="radio" name="member" value="others" id="select" name="radioButton">
+                  others
+                </label>
+              </div>
+            </div>
+          </div>                   
+        </div>
+        <div id="error-show"></div>
+        <div class="field">
+          <div class="control">
+            <label class="checkbox">
+              <input type="checkbox" name="checkbox">
+              I agree to the <a href="#">terms and conditions</a>
+            </label>
+          </div>
+        </div>               
+      </div>
+    </div>
+  <br><br>
+        <div class="field">
+          <p class="control">
+            <button class="button is-success" id="submit" onclick="myfunction()">
```

Comment:
If you developing in Jquery, use jquery everywhere. Here `onclick` is javascript event. There is no change in output if you use onclick. But it is not good to use jquery in some places and javascript in other places

Reply:
but click event in jquery not working for display values,how can i get values

Reply:
Find out. I'm sure there is way to do that.